src/components/Flags/En.js

```diff
@@ -6,11 +6,11 @@ const En = (props) => {
     <Icon viewBox="0 0 640 480" {...props}>
     <defs>
    <clipPath id="a">
-     <path fill-opacity=".7" d="M-85.3 0h682.6v512H-85.3z"/>
+     <path fillOpacity=".7" d="M-85.3 0h682.6v512H-85.3z"/>
```

Comment:
i get an error with these svg attributes: fill-opacity, clip-path, stroke-width, fill-rule, respectively to change in fillOpacity, clipPath, strokeWidth, fillRule